README.md

```diff
@@ -1,21 +1,58 @@
 # osl-repos
 
-TODO: Enter the cookbook description here.
-
+  This cookbook manages the base, epel, extras, and updates repositories on Centos 7; and the appstream, base, elrepo,
+  epel, and powertools repositories on Centos 8. This cookbook imports these repositories and points them at the osuosl
+  repositories.
 ## Requirements
 
+ - Chef/Cinc 15+
+ - Centos 7+
 ### Platforms
 
 - CentOS 7+
 
 ### Cookbooks
 
+ - yum
+ - yum-centos
+ - yum-elrepo
+ - yum-epel
 ## Attributes
 
+  None
+
 ## Resources
 
+  osl_repos_centos
```

Comment:
I think I would feel better if this was split into three separate resources so that there's no confusion:
- `osl_repos_centos` (yum-centos only)
- `osl_repos_epel` (yum-epel only)
- `osl_repos_elrepo` (yum-elrepo only)

Does that make sense?